### PR TITLE
chore: rename ceresmeta to horaemeta

### PR DESCRIPTION
**Pull request by chunshao90**
## Rationale
Rename ceresdb to horaedb in the `Readme` and  `go.mod`.

## Detailed Changes
Rename ceresdb to horaedb in the `Readme` and  `go.mod`.

## Test Plan
CI.

### CODE REVIEW

File: server/config/config.go [hunk 1]
```diff
@@ -54,13 +54,13 @@ const (
 	// GrpcServiceKeepAlivePingMinIntervalSec controls the min interval for one keepalive ping.
 	defaultGrpcServiceKeepAlivePingMinIntervalSec int = 20
 
-	defaultNodeNamePrefix          = "ceresmeta"
+	defaultNodeNamePrefix          = "horaemeta"
```
Comment (ZuLiangWang) [inline server/config/config.go:57]: There may be a compatibility issue here.

File: server/config/config.go [hunk 2]
```diff
@@ -54,13 +54,13 @@ const (
...
 	defaultEndpoint                = "127.0.0.1"
 	defaultRootPath                = "/ceresdb"
 	defaultClientUrls              = "http://0.0.0.0:2379"
 	defaultPeerUrls                = "http://0.0.0.0:2380"
 	defaultInitialClusterState     = embed.ClusterStateFlagNew
-	defaultInitialClusterToken     = "ceresmeta-cluster" //#nosec G101
+	defaultInitialClusterToken     = "horaemeta-cluster" //#nosec G101
```
Comment (ZuLiangWang) [inline server/config/config.go:63]: There may be a compatibility issue here.

File: server/config/config.go [hunk 3]
```diff
@@ -79,14 +79,14 @@ const (
...
-	// topologyType is used to determine the scheduling cluster strategy of CeresMeta. It should be determined according to the storage method of CeresDB. The default is static to support local storage.
+	// topologyType is used to determine the scheduling cluster strategy of HoraeMeta. It should be determined according to the storage method of HoraeDB. The default is static to support local storage.
 	defaultTopologyType                = "static"
 	defaultProcedureExecutingBatchSize = math.MaxUint32
 
 	defaultHTTPPort = 8080
 	defaultGrpcPort = 2379
 
-	defaultDataDir = "/tmp/ceresmeta"
+	defaultDataDir = "/tmp/meta"
```
Comment (ZuLiangWang) [inline server/config/config.go:89]: Reset this line.

File: server/config/config.go (hunk 2 above)
Comment (chunshao90) [reply server/config/config.go:63]: Only the default cluster name has been changed, theoretically there should be no compatibility issues.

File: server/config/config.go (hunk 1 above)
Comment (chunshao90) [reply server/config/config.go:57]: During the meta upgrade process, the new and old nodes are unable to discover each other, but it does not affect the overall functionality.

**Review by ZuLiangWang — APPROVED** [review]
LGTM

**Pull request merged**